backend/demo/src/main/java/com/snacks/demo/response/ResponseService.java

```diff
@@ -33,17 +33,15 @@ public<T> ListResponse<T> getListResponse(T datalist) {
     }
 
     //오류를 반환할때
-    public CommonResponse errorResponse(int status, String log)
+    public CommonResponse errorResponse(String log)
     {
         CommonResponse errorResponse = new CommonResponse();
```

Comment:
```suggestion
var errorResponse = new CommonResponse();
```
이런 식으로 써도 괜찮을 것 같습니다